bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/GateAddressApi.kt

```diff
@@ -145,4 +145,20 @@ interface GateAddressApi {
     @PutExchange("/output/addresses")
     fun putAddressesOutput(@RequestBody addresses: Collection<AddressGateOutputRequest>): ResponseEntity<Unit>
 
+
+    @Operation(
+        summary = "Get address csv file",
+        description = "Get address csv file."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Address CSV file was successfully provided"),
+            ApiResponse(responseCode = "404", description = "Address CSV file was not successful", content = [Content()])
+        ]
+    )
+
+    @GetMapping("/input/addresses/csv")
+    @GetExchange("/input/addresses/csv")
```

Comment:
Better Practice for REST Apis: Let's offer the CSV over the same POST addresses/search signature but change the response content type to "text/csv". So the endpoint can produce its results in JSON and in CSV depending on what content the user wants